pkgs/misc/vscode-extensions/default.nix

```diff
@@ -189,6 +189,23 @@ let
         };
       };
 
+      elmtooling.elm-ls-vscode = buildVscodeMarketplaceExtension {
+        meta = with lib; {
+          changelog = "https://marketplace.visualstudio.com/items/Elmtooling.elm-ls-vscode/changelog";
+          description = "Elm language server";
+          downloadPage = "https://marketplace.visualstudio.com/items?itemName=Elmtooling.elm-ls-vscode";
+          homepage = "https://github.com/elm-tooling/elm-language-client-vscode";
+          license = licenses.mit;
+          maintainers = with maintainers; [ mcwitt ];
```

Comment:
I know a lot of extensions do it wrong here but generally we want to have meta last in a derivation.

Reply:
@SuperSandro2000 thanks, fixed!